DESCRIPTION

```diff
@@ -24,11 +24,18 @@ Authors@R: c(
           family = "Sequeira",
```

Comment:
Remember to change version number

Reply:
Could we try to have as many version numbers for the dependancies as possible/reasonable?

Also, include any changes to R version or dependancies in the Changelog. So, for the changelog for this release, include the version for R and all dependancies (or at least the ones we know). 

This way, people can quickly check the changeling and know what version of cchsflow they can use.

Reply:
Make sense?